api/server.spec.js

```diff
@@ -0,0 +1,61 @@
+const request = require("supertest");
+const server = require("./server");
+// const auth = require("./server");
+describe('index route', () => {
+  it('should return an OK status code from the index route', async () => {
+    const expectedStatusCode = 200;
+    const response = await request(server).get('/');
+    expect(response.status).toEqual(expectedStatusCode);
+  });
+});
+
+// beforeEach(async () => {
+//   // this function executes and clears out the table before each test
+//   await db('test').truncate();
+// });
+//beforeEach reset is broken, so be sure to make a new username in testee when you want this to run
+const testee = {
+  "username": "RealBoy17", "password": "Pinocchio"
+}
+const failureTestee = {
+  "username": "FakeBoy",
+}
+
+describe("auth-router.js", () => {
```

Comment:
I would put this spec in its own file, next to the router it is testing. You could make a simple test app that only mounts this router too, that way any middleware (such as the auth middleware in front of the jokes endpoint) could be ignored in these tests.